client/package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^13.4.0",
```

Comment:
Double check what should be a dev dependency and what should be a prod dependency: https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencie